frame/elections-phragmen/src/lib.rs

```diff
@@ -1057,14 +1032,14 @@ impl<T: Trait> Module<T> {
 			<RunnersUp<T>>::put(new_runners_up);
 
 			Self::deposit_event(RawEvent::NewTerm(new_members.clone().to_vec()));
-		} else {
-			Self::deposit_event(RawEvent::EmptyTerm);
-		}
 
-		// clean candidates.
-		<Candidates<T>>::kill();
+			// clean candidates.
+			<Candidates<T>>::kill();
 
-		ElectionRounds::mutate(|v| *v += 1);
+			ElectionRounds::mutate(|v| *v += 1);
+		}).map_err(|_| {
```

Comment:
the call to seq-phramen here can only fail if the internal `normalize` fails.. If we are to be a bit more risky, we can `expect` here. It is proven the `seq_phragmen` with `Perbill` and maximum 16 votes per voter will never fail.